src/_guides/language/effective-dart/usage.md

```diff
@@ -721,22 +720,32 @@ then. That always happens after all initializers are run regardless of where
 useful. In fact, [dartdevc]({{site.webdev}}/tools/dartdevc)
 *requires* that it appear last.
 
-<div class="good">
+{% comment %}
+Question for Kathy: I've added a "Dart 2.0 note" below.
+Do we need to keep the last sentence above?
+{% endcomment %}
+
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (super-first)"?>
 {% prettify dart %}
 View(Style style, List children)
     : _children = children,
-      super(style) {
+      super(style);
 {% endprettify %}
-</div>
 
-<div class="bad">
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (super-first)"?>
 {% prettify dart %}
 View(Style style, List children)
-    : super(style),
-      _children = children {
+    : super(style), // Error in Dart 2.0
+      _children = children;
 {% endprettify %}
-</div>
 
+{% comment %}update-for-dart-2{% endcomment %}
+<div class="alert alert-danger" markdown="1">
+**Dart 2.0 note:** The call to `super()` must be _last_ in Dart 2.0, otherwise
+an error is reported.
+</div>
```

Comment:
@kwalrath - I've added a Dart 2.0 note here. Is this ok?

Reply:
I like the note, but I'd emphasize _must_ instead of "last", or maybe _must be last_.

Reply:
I had previously only emphasized _must_. Ok, let's emphasize "_must be last_".